### PR TITLE
Yaml output conifg

### PR DESCRIPTION
**Pull request by aradoje**

### CODE REVIEW

File: symfony/index.rst [hunk 1]
```diff
@@ -91,6 +91,7 @@ must implement ``Translation\Common\Storage``.
           output_dir: "%kernel.root_dir%/Resources/translations"
           remote_storage: ["php_translation.adapter.loco"]
           local_storage: ["app.custom_local_storage"]
+          output_format: "yml"
```
Comment (Nyholm) [inline symfony/index.rst]: Make it "xlf" because it is the default value. 

Also, xlf is the internet standard format for translation files.

**Review by Nyholm — CHANGES_REQUESTED** [review]
Thank you for adding this.

**Comment by aradoje** [comment]
My pleasure.

**Review by Nyholm — APPROVED** [review]
Thanks

**Pull request merged**